apps/demo-game/src/pages/admin/reports/[id].tsx

```
@@ -0,0 +1,458 @@
+        })
+      })
+  })
+  const segmentResultPerPlayerAvg = segmentResultsPerPlayer.map((arr) => {
+    return arr.reduce((acc, val, ix) => {
+      val.map((v, i) => (acc[i] += v / arr.length))
+      return acc
+    }, Array(arr[0].length).fill(0))
+  })
+
+  totalDecisionAvg = segmentResultPerPlayerAvg.reduce((acc, val) => {
+    val.map((v, i) => (acc[i] += v / segmentResultPerPlayerAvg.length))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)